api-gateway/src/main/kotlin/com/saveourtool/save/gateway/security/WebSecurityConfig.kt

```diff
@@ -128,7 +128,7 @@ class WebSecurityConfig(
             it.authenticationSuccessHandler(
                 DelegatingServerAuthenticationSuccessHandler(
                     StoringServerAuthenticationSuccessHandler(configurationProperties),
-                    RedirectServerAuthenticationSuccessHandler("/#/projects"),
+                    RedirectServerAuthenticationSuccessHandler("/#/registration"),
```

Comment:
Will it work only when it is the first time that the user is authenticated with his GitHub account?

Reply:
Yes. We check if the user is registered. If so, move it to _projects_ page

Reply:
But why do we need to open registration page every time? We should distinguish fisrt-time users from already registered ones

Reply:
Yes, on the registration view there is a check that the user is registered. And redirect the user to the project view even before rendering.

Reply:
https://github.com/saveourtool/save-cloud/pull/965#discussion_r953695679